### ISSUE TITLE
fix the bug that the pgxc_ctl parameter "walArchive" doessn't take effect during init

### ISSUE THREAD

**Issue by xuegang** [postgres-x2/postgres-x2#155]
fix the bug that the pgxc_ctl parameter "walArchive" doessn't take effect during init.
The current mode is 
 if (datanodeSlave=y)
 {
     if (walArchive = y ) do archive
     else doesn't do archive
 }
 else if (datanodeSlave=n)
 {
    walArchive takes no effect.

Later, I will send patch to you.

**Comment by koichi-szk** [comment]
Yes, it is at the begging of cascaded/multiple slave setups.   I eventually found the spec was not good.   I think we need some more careful design for this.   Until then, if there's no significant side-effect, I'd like to leave this as is.

Any other thoughts?

**Comment by koichi-szk** [comment]
Would like to resolve this when we allow more than one slave and cascaded slave.
